internal/super/flix.go

```diff
@@ -121,6 +129,7 @@ func execute(
 		}
 
 	case flixPath:
+		isLocal = true
 		file, err := os.ReadFile(flixQuery)
```

Comment:
use state.ReadFile here since it uses the underlying ReaderWrite. 

(yes i know this line is not in this PR but it should be used)